src/Features/Core/Portable/Diagnostics/Analyzers/NamingStyleDiagnosticAnalyzerBase.cs

```diff
@@ -40,7 +39,7 @@ protected NamingStyleDiagnosticAnalyzerBase()
         // see https://github.com/dotnet/roslyn/issues/14061
         protected abstract ImmutableArray<TLanguageKindEnum> SupportedSyntaxKinds { get; }
 
-        public override bool OpenFileOnly(Workspace workspace) => true;
```

Comment:
do we know why it was open file only before? there used to be 2 reasons as long as I know.

first is perf. some analyzer was just so slow, so making it to run all files just make full solution analysis useless. so it was set to open file only until perf is improved.

second was it required a workspace/vs service which doesn't exist in OOP including some options.

Reply:
➡️ The changes to `OpenFileOnly` methods were reverted (no other changes so I amended the previous commit)